slstatus.c

```diff
@@ -759,7 +759,7 @@ int
 main(int argc, char *argv[])
 {
 	unsigned short int i;
-	char status_string[4096];
+	char status_string[2048];
```

Comment:
not really, a status bar on a 1920x1080 monitor would not even need that 2048, see: http://hexeract.info/pub/len.png